backend/src/backend/user_session/routers/general.py

```diff
@@ -10,11 +10,9 @@
 START_TIME_CONTAINER = datetime.datetime.now()
 
 
-def human_readable(psutil_object: NamedTuple) -> Dict[str, Union[str, Dict[str, str]]]:
+def _convert_psutil_object(psutil_object: NamedTuple) -> Dict[str, Union[str, Dict[str, str]]]:
     return {
-        key: f"{getattr(psutil_object, key):.1f} %"
-        if key == "percent"
-        else f"{getattr(psutil_object, key) / (1024**3):.2f} GiB"
+        key: getattr(psutil_object, key) if key == "percent" else getattr(psutil_object, key)
```

Comment:
I haven't programmed in Python for a while, but this if statement seems to return the same thing no matter what?